File: food-order/app/entrypoint/rest/order_response.go
package rest

type OrderResponse struct {
	Uuid       string                `json:"uuid"`
	UserUuid   string                `json:"user_uuid"`
    Number     int                   `json:"number"`
    Status     string                `json:"status"`
	Restaurant RestaurantResponse    `json:"restaurant"`
    Payment    *PaymentOrderResponse `json:"payment,omitempty"`
	Items      []OrderItemResponse   `json:"items"`
	Total      float64               `json:"total"`
}

type OrderItemResponse struct {
	Uuid         string  `json:"uuid"`
	MenuItemUuid string  `json:"menu_item_uuid"`
	Name         string  `json:"name"`
	Amount       int     `json:"amount"`
	UnitValue    float64 `json:"unitValue"`
}

type RestaurantResponse struct {
	Uuid string `json:"uuid"`
	Name string `json:"name"`
}

type PaymentOrderResponse struct {
    Uuid          string `json:"uuid"`
    Status        string `json:"status"`
    TransactionId string `json:"transaction_id"`
}

File: food-order/Entrypoint/Rest/Json/OrderResponse.cs
using System.Collections.Generic;

namespace food_order.Entrypoint.Rest.Json
{
    public class OrderResponse
    {
        public string Uuid { get; set; }
        public RestaurantResponse Restaurant { get; set; }
        public List<OrderItemResponse> Items { get; set; }
        public decimal Total { get; set; }

        public OrderResponse(string uuid, RestaurantResponse restaurant, 
            List<OrderItemResponse> items, decimal total) 
        {
          this.Uuid = uuid;
          this.Restaurant = restaurant;
          this.Items = items;
          this.Total = total;
        }
    }
}
